### PR TITLE
Add options for workflows

### PR DESCRIPTION
**Pull request by gerrod3**
My docs work in https://github.com/pulp/pulp_gem/pull/108 requires these changes to the CLI.

### CODE REVIEW

**Comment by mdellweg** [comment]
Do you intend to add tests?

**Comment by gerrod3** [comment]
> Do you intend to add tests?

Yes

File: tests/scripts/pulp_gem/test_remote.sh [hunk 1]
```diff
@@ -12,9 +12,9 @@ trap cleanup EXIT
 
 expect_succ pulp gem remote list
 
-expect_succ pulp gem remote create --name "cli_test_gem_remote" --url "$GEM_REMOTE_URL"
+expect_succ pulp gem remote create --name "cli_test_gem_remote" --url "$GEM_REMOTE_URL" --includes '{"panda":null}'
 
-expect_succ pulp gem remote update --remote "cli_test_gem_remote" --policy "on_demand"
+expect_succ pulp gem remote update --remote "cli_test_gem_remote" --policy "on_demand" --prereleases "True"
```
Comment (mdellweg) [inline tests/scripts/pulp_gem/test_remote.sh:17]: We can either set, reset or ignore this boolean flag, right? Would we get this tristate from click if we made it a `"--prereleases/--no-prereleases"` flag?

**Review by mdellweg — DISMISSED** [review]

File: tests/scripts/pulp_gem/test_remote.sh (hunk 1 above)
Comment (gerrod3) [reply tests/scripts/pulp_gem/test_remote.sh:17]: Let me test.

Comment (gerrod3) [reply tests/scripts/pulp_gem/test_remote.sh]: Yep, it does work as a tristate when using the flag option.

**Review by mdellweg — APPROVED** [review]

**Comment by mdellweg** [comment]
Can you run Black on this? I want to get it merged. ;)

**Review by mdellweg — DISMISSED** [review]

**Review by mdellweg — APPROVED** [review]

**Pull request merged**